pandas/tests/frame/test_missing.py

```diff
@@ -465,6 +465,36 @@ def test_frame_fillna_limit(self):
         expected.values[:3] = np.nan
         tm.assert_frame_equal(result, expected)
 
+    def test_frame_fillna_axis(self):
```

Comment:
do we have a similar test for axis=0? ideally you can just parametrize those tests